.gitignore

```diff
@@ -0,0 +1,3 @@
+#this file will ignore 
+test.md
+module 
```

Comment:
- [ ] Kindly be noticed that you should write `node_module` instead of `module`.